### PR TITLE
Microsoft.Bicep version 0.6.1

### PR DESCRIPTION
**Pull request by frenky29a**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59622)

### CODE REVIEW

File: manifests/m/Microsoft/Bicep/0.6.1/Microsoft.Bicep.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.Bicep
+PackageVersion: 0.6.1
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/Bicep/0.6.1/Microsoft.Bicep.installer.yaml:5]:
```suggestion
PackageVersion: 0.6.1.6515
```
This must match Control Panel.

You will also need to change the version folder from `0.6.1` to `0.6.1.6515`

File: manifests/m/Microsoft/Bicep/0.6.1/Microsoft.Bicep.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.Bicep
+PackageVersion: 0.6.1
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/Bicep/0.6.1/Microsoft.Bicep.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.6.1.6515
```
This must match Control Panel.

You will also need to change the version folder from `0.6.1` to `0.6.1.6515`

File: manifests/m/Microsoft/Bicep/0.6.1/Microsoft.Bicep.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.Bicep
+PackageVersion: 0.6.1
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/Bicep/0.6.1/Microsoft.Bicep.yaml:5]:
```suggestion
PackageVersion: 0.6.1.6515
```
This must match Control Panel.

You will also need to change the version folder from `0.6.1` to `0.6.1.6515`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by frenky29a** [comment]
Thanks [ItzLevvie](https://github.com/ItzLevvie) for version comment. I'll submit corrected one.

**Pull request closed without merging**